### PR TITLE
Update changelog for version 0.38.2

### PR DESCRIPTION
**Pull request by 4383**
Related to #1007

### CODE REVIEW

**Review by itamarst — APPROVED** [review]
Thank you!

**Comment by 4383** [comment]
Ok, then let's pause the release process.

File: NEWS [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Unreleased
 ==========
 
+0.39.0
+======
+
+* [fix] fix the monkey patching with the asyncio hub
```
Comment (itamarst) [inline NEWS:7]: Really it's only the first bullet item, I think.

Comment (itamarst) [reply NEWS:7]: Second one is implementation detail, third one isn't true anymore because I came up with better unmonkeypatching.

File: NEWS [hunk 2]
```diff
@@ -1,6 +1,13 @@
 Unreleased
 ==========
 
+0.39.0
```
Comment (itamarst) [inline NEWS:4]: Yeah 0.38.1 is more accurate

**Review by itamarst — APPROVED** [review]
Couple of comments to address, and then can merge, thank you!

File: NEWS (hunk 2 above)
Comment (4383) [reply NEWS:4]: 0.38.2 because 0.38.1 is already out

**Comment by 4383** [comment]
done

**Comment by 4383** [comment]
(I didn't changed the name of the branch used to create the pull request, it still reflect the previous version proposal, but, eh, who care? :smile: )

**Comment by itamarst** [comment]
Looks like 3.7 can't run on Ubuntu 24.04.

**Pull request merged**